setup.py

```diff
@@ -13,11 +13,6 @@
           'requests==2.22.0',
           'singer-python==5.8.1'
       ],
-      extras_require={
-          'dev': [
-              'pylint'
-          ]
-      },
```

Comment:
I think we should add this back in at some point. I think ignoring this in alpha is fine.

Reply:
Ok - added back.